app/serializers/spree/v2/storefront/vendor_image_serializer.rb

```diff
@@ -0,0 +1,11 @@
+module Spree
+  module V2
+    module Storefront
+      class VendorImageSerializer < BaseSerializer
+        set_type :vendor_image
+
+        attributes :viewable_type, :viewable_id, :styles
```

Comment:
@poudelprakash we can skip type/id

```suggestion
        attributes :styles
```